WelsonJS.Toolkit/WelsonJS.Launcher/AssemblyLoader.cs

```diff
@@ -365,16 +381,19 @@ private static void DownloadFile(string url, string dest)
         }
 
 
-        private static bool TryDownloadGzipToFile(string gzUrl, string dest)
+        private static bool TryDownloadCompressedFile(string gzUrl, string dest)
         {
             string tempFile = dest + ".tmp";
 
             try
             {
-                using (var res = Http.GetAsync(gzUrl).GetAwaiter().GetResult())
+                using (var res = LegacyHttp.GetAsync(gzUrl).GetAwaiter().GetResult())
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential SSRF (Server-Side Request Forgery) vulnerability detected. This pattern identifies code structures that might allow attackers to manipulate server-side requests, posing a risk of unauthorized access and data leaks. Ensure that user-controlled inputs in HttpClient, WebClient, or RestClient requests are thoroughly validated and sanitized to mitigate this security threat.